SystemServices/SystemServices.cpp

```diff
@@ -67,7 +67,7 @@ using namespace std;
 
 #define API_VERSION_NUMBER_MAJOR 1
 #define API_VERSION_NUMBER_MINOR 7
-#define API_VERSION_NUMBER_PATCH 0
+#define API_VERSION_NUMBER_PATCH 2
```

Comment:
Pls increment major version (2.0.0) since we are removing API. You would FYI refer to https://github.com/rdkcentral/RDKServices#versioning

Reply:
Updated.